examples/update_flow_control/update_flow_control.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""example application to control the Update flow of the Mender client
```

Comment:
```suggestion
#
# example application to control the Update flow of the Mender client
```

Reply:
Mmmm. But what I am trying to achieve with this is to have a module level docstring (pylint will complain if I don't)

Reply:
Oh, I see :)
All good then